src/utils/utils.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * General utilities.
+ *
+ * @package CoBlocks
+ */
+
+/**
+ * This function allows to easily access a part from a php array.
+ * It is equivalent to want lodash get provides for JavaScript and is useful to have something similar
+ * in php so functions that do the same thing on the client and sever can have identical code.
+ * 
+ * Utility is cloned from source: 
+ * https://github.com/WordPress/gutenberg/blob/5f614054e92f5147018f07db601ac1c5eae9d14e/lib/utils.php#L8-L33
+ *
+ * @param array $array    An array from where we want to retrieve some information from.
+ * @param array $path     An array containing the path we want to retrieve.
+ * @param array $default  The return value if $array or $path is not expected input type.
+ *
+ * @return array An array matching the path specified.
+ */
+function coblocks_experimental_get( $array, $path, $default = array() ) {
```

Comment:
@AnthonyLedesma Any reason why we couldn't just use the Gutenberg function directly?